src/CommandLineUtils/McMaster.Extensions.CommandLineUtils.csproj

```diff
@@ -33,5 +33,15 @@ McMaster.Extensions.CommandLineUtils.ArgumentEscaper
     <PackageReference Include="System.Threading.Thread" Version="4.0.0" />
     <PackageReference Update="NETStandard.Library" PrivateAssets="All" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <DotNetCliToolReference Include="dotnet-sourcelink-git" Version="2.8.0" />
```

Comment:
I'm not familiar with these tools. Are they necessary? Is there another way to add SourceLink support? I'd rather not add DotNetCliToolReference tools if it can be avoided.

Reply:
It is required if you want to test that the files referenced in the PDB match what's in GitHub.  I set this to happen when $IsOfficialBuild is true.  If that's not something you are interested in, I'd be happy to pull it out and use the SourceLink.Create.CommandLine which has no dependency on DotNetCliToolReferences.

Reply:
Yeah let’s go with that option. 

Btw, does this change package output to include the PDBs automatically?

Reply:
I believe it does automatically include the PDBs when [publishing](https://github.com/ctaggart/SourceLink#package-pdb-in-nupkg).